src/OnchainKitProvider.tsx

```diff
@@ -27,13 +28,15 @@ export function OnchainKitProvider({
       address: address ?? null,
       apiKey: apiKey ?? null,
       chain: chain,
+      config: config ?? null,
       projectId: projectId ?? null,
       rpcUrl: rpcUrl ?? null,
       schemaId: schemaId ?? null,
     };
     setOnchainKitConfig(onchainKitConfig);
     return onchainKitConfig;
-  }, [address, apiKey, chain, projectId, rpcUrl, schemaId]);
+  }, [address, apiKey, chain, config, projectId, rpcUrl, schemaId]);
+
```

Comment:
thoughts on adding a useTheme hook instead of accessing through useOnchainKit?   I'm on the fence with that, but throwing it out there.

Reply:
I feel it makes sense to pass the `config` here because the users are passing the parameters directly in the `OnchainKitProvider`.

Reply:
Open to using a useTheme hook if you feel that it would be better!